src/js/redux/reducers/search/filters/locationFilterFunctions.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Created by michaelbray on 12/12/16.
+ */
+
+import { Set } from 'immutable';
+
+const updateSelectedLocations = (state, value, direction) => {
+    let updatedSet = new Set(state);
+
+    if (updatedSet.includes(value) && direction === 'remove') {
```

Comment:
In theory, I think we can do that. I had copied this functionality from `bulkAwardTypeChange` in `awardFilterFunctions.js`.